### PR TITLE
Add terminal font size change on rows change

### PR DESCRIPTION
**Pull request by sabbaka**
Here is result, unfortunately doing calculations on fly is too big performance hit, so I've ended up providing static values. Also, I have saved my functions, so could use them in future.
Closes #64 #30

### CODE REVIEW

File: pkg/session_recording/player.jsx [hunk 1]
```diff
@@ -620,6 +620,229 @@
...
+                            409: 0.20000000000000007,
+                            414: 0.20000000000000007,
+                            418: 0.20000000000000007,
+                            424: 0.20000000000000007,
+                            423: 0.20000000000000007,
+                            422: 0.20000000000000007,
+                            421: 0.20000000000000007,
+                            419: 0.20000000000000007,
+                            420: 0.20000000000000007,
+                        };
```
Comment (spbnick) [inline pkg/session_recording/player.jsx]: Could you please put this outside the function and class? Also, these seem to contain only a few significant digits. You can truncate them to just two digits after the point. E.g. 0.20, 0.29, 0.37, etc.

File: pkg/session_recording/player.jsx [hunk 2]
```diff
@@ -620,6 +620,229 @@
...
+
+                        document.getElementsByClassName('console-ct')[0].style.fontSize = relation_pixels[pkt.width] + "em";
```
Comment (spbnick) [inline pkg/session_recording/player.jsx]: What happens if the height is changed?

**Review by spbnick — CHANGES_REQUESTED** [review]

File: pkg/session_recording/player.jsx (hunk 2 above)
Comment (sabbaka) [reply pkg/session_recording/player.jsx]: Right, we want to keep height same also, will add this.

File: pkg/session_recording/player.jsx (hunk 1 above)
Comment (sabbaka) [reply pkg/session_recording/player.jsx]: Yes, sure, will do. Have you tried how it's working? Do you have any comments?

**Comment by spbnick** [comment]
Well, I just tried and it behaved nothing like I expected. I'll wait for full resize support :)

**Pull request closed without merging**